.ci/docker/ubuntu/Dockerfile

```diff
@@ -175,7 +175,7 @@ RUN if [ -n "${SKIP_LLVM_SRC_BUILD_INSTALL}" ]; then set -eu; rm -rf /opt/llvm;
 
 # AWS specific CUDA build guidance
 ENV TORCH_CUDA_ARCH_LIST Maxwell
-ENV TORCH_NVCC_FLAGS "-Xfatbin -compress-all"
+ENV TORCH_NVCC_FLAGS "-Xfatbin -compress-all -compress-mode=size"
```

Comment:
Aren't whl a zip file already? If that's the case, shoudl we try to package it with -9 or something and achieve the same result?

Reply:
We had -9 flag for zipping wheels before uploading to pypi for a while. This have not produced noticeable difference: 
https://github.com/pytorch/test-infra/blob/b4870dd25914dbc61267c945726717f74207c0ef/release/pypi/prep_binary_for_pypi.sh#L77

Reply:
This users a more aggressive compression BEFORE it gets compressed by fatbin. so it compresses better then trying to run aggressive compression on top of mild compression.

Reply:
Presumably it's zlib, but it's not clear. They used to use fatbin I think, but switched to nvfatbin recently, not sure it's open source. https://docs.nvidia.com/cuda/nvfatbin/index.html